vagrant/plugins/vagrant-hypconfigmgmt/Makefile

```diff
@@ -10,7 +10,7 @@ all:
 test:
 	bundle exec rspec spec/
 install:
-	find pkg/ -name '*.gem' | head -n 1 | xargs vagrant plugin install
+	find pkg/ -name '*.gem' | tail -n 1 | xargs vagrant plugin install
```

Comment:
Because otherwise `make install` always installs the oldest version:

```
vdloo@workstation4:~/code/projects/hypernode-vagrant/vagrant/plugins/vagrant-hypconfigmgmt$ make
rake build
vagrant-hypconfigmgmt 0.0.6 built to pkg/vagrant-hypconfigmgmt-0.0.6.gem.
vdloo@workstation4:~/code/projects/hypernode-vagrant/vagrant/plugins/vagrant-hypconfigmgmt$ ls pkg/
vagrant-hypconfigmgmt-0.0.5.gem  vagrant-hypconfigmgmt-0.0.6.gem
vdloo@workstation4:~/code/projects/hypernode-vagrant/vagrant/plugins/vagrant-hypconfigmgmt$ find pkg/ -name '*.gem' | tail -n 1
pkg/vagrant-hypconfigmgmt-0.0.6.gem
vdloo@workstation4:~/code/projects/hypernode-vagrant/vagrant/plugins/vagrant-hypconfigmgmt$ find pkg/ -name '*.gem' | head -n 1
pkg/vagrant-hypconfigmgmt-0.0.5.gem
```